client/my-sites/current-site/index.jsx

```diff
@@ -152,7 +93,8 @@ class CurrentSite extends Component {
 					</div>
 					: <AllSites />
 				}
-				{ ! isJetpack && this.getDomainWarnings() }
+				<AsyncLoad require="my-sites/current-site/domain-warnings"
+					placeholder={ <EmptyComponent /> } />
```

Comment:
Should this be accomplished differently? In other places we are displaying none the placeholder. Maybe a prop in `AsyncLoad` called `invisible` or similar.